src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateView.component.tsx

```diff
@@ -296,16 +313,153 @@ export const renderEditorHeading = (
                         className={`cursor ${overridden ? 'cr-5' : 'cb-5'}`}
                         onClick={handleOverride}
                     >
-                        {overridden ? 'Delete override' : 'Allow override'}
+                        {isValues?(overridden ? 'Delete override' : 'Allow override'):''}
                     </span>
                 )}
             </div>
         </div>
     )
-}
 
-export const SuccessToastBody = ({ chartConfig }) => {
+export const CompareWithApprovalPendingAndDraft = ({
+    isEnvOverride,
+    overridden,
+    readOnly,
+    environmentName,
+    selectedChart,
+    handleOverride,
+    latestDraft,
+    isPublishedOverriden,
+    isDeleteDraftState,
+    setShowDraftData,
+    isValues
+}: CompareWithApprovalPendingAndDraftProps) => {
+    const [selectedOption, setSelectedOption] = useState({ id: 0, label: 'Approval Pending' })
+
+    const options = [
+        {
+            label: 'Manifest generated from',
+            options: [
+                { id: 0, label: 'Approval Pending' },
```

Comment:
We should create const for this option as it is used on multiple places

Reply:
done